tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+# Copyright (c) 2019-2020, BlazingSQL, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+
+
+from blazingsql import BlazingContext
+from xbb_tools.cluster_startup import attach_to_cluster
+from dask_cuda import LocalCUDACluster
+from dask.distributed import Client
```

Comment:
Looks like we can remove these unused imports

Reply:
Removed.